web/styles/inputs.css

```diff
@@ -0,0 +1,86 @@
+.input-element-wrapper {
+    display: grid;
+    grid-template: "icon input button" auto / auto minmax(0, 1fr) auto;
+    grid-template-rows: 1fr;
```

Comment:
Why is the row value `1fr` instead of `auto`?

Reply:
Especially since you have `auto` already set on the `grid-template` property above...

Reply:
Makes sense, I was confused for some time there with the syntax. Anyway's here is the updated grid-template-definition:
```css
    grid-template:
        [input-element-start] "icon-left-margin input-icon icon-content-gap content button-content-gap input-button button-right-margin" auto [input-element-end] /
        [input-element-start] var(--input-icon-left-margin) var(--input-icon-width) var(--input-icon-content-gap) minmax(0, 1fr) var(--input-button-content-gap) var(--input-button-width) var(--input-button-right-margin) [input-element-end];
```